sites/theme.scss

```diff
@@ -367,6 +368,8 @@
 	@import "search/screen-xs-min-to-screen-xs-max";
 
 	@import "../components/header-rwd/screen-xs-min-to-screen-xs-max";
+
+	@import "../templates/application/screen-xs-max";
```

Comment:
This import are not inserted in the right location. For example it is expected to work in `.xxsmallview`.

I suggest to move it near line 233